src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,13 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
same here

try to read when we need to use fragment https://react.dev/reference/react/Fragment